SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -280,6 +275,25 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
   AD::EndNoSharedReading();
 }
 
+void CTurbSSTSolver::Viscous_Residual(unsigned long iEdge, CGeometry* geometry, CSolver** solver_container,
+                                     CNumerics* numerics, CConfig* config) {
+
+  /*--- Define an object to set solver specific numerics contribution. ---*/
+  struct SolverSpecificNumerics {
+
+    FORCEINLINE void operator() (const CGeometry& geometry, CNumerics& numerics, CConfig& config, const CScalarVariable& nodes, unsigned long iPoint, unsigned long jPoint) const {
+      /*--- Menter's first blending function (only SST)---*/
+      if ((config.GetKind_Turb_Model() == SST) || (config.GetKind_Turb_Model() == SST_SUST))
+        numerics.SetF1blending(nodes.GetF1blending(iPoint), nodes.GetF1blending(jPoint));
```

Comment:
```suggestion
      numerics.SetF1blending(nodes.GetF1blending(iPoint), nodes.GetF1blending(jPoint));
```

Reply:
also done in https://github.com/su2code/SU2/pull/1330/commits/3e86076ad4641710db97f5aa38cd23af11b419ae